### DISCUSSION TITLE
Trailling lambda

### DISCUSSION THREAD

**Discussion by UseCser** [dotnet/csharplang#8437] [Language Ideas]
We can reference something features from other language， like kotlin
I love it(Trailing lambda) very much, and it is convenient for using 
In C# we can use property like this:
`
public class C
{
    public Int32 Age {get; set;}
}
`
the property "Age" in java is used to writing a function, but in C# we can omit parentheses
In kotlin if the last parameter is lambda and we also can omit parentheses
[https://kotlinlang.org/docs/lambdas.html#passing-trailing-lambdas](url)
`
fun f(v:()->Unit) {xxx}
fun main(){
   f{
      xxxx
   }
}
`
Following this:
In c# 
`
F(){}
void F(Action a){}
`
In my idea we needn't omit parentheses because it will make diffrent meanings

**Comment by HaloFour** [comment]
See: #6122 although that discussion is currently locked

**Reply by UseCser** [reply]
thx, The same function in #1151
